app/apollo/resolvers/channel.js

```diff
@@ -405,6 +405,9 @@ const channelResolvers = {
           }
         } ) );
 
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {name: newChannelObj.name, uuid: newChannelObj.uuid, versions: versions, subscriptions: subscriptions};
```

Comment:
instead of dumping the entire subscriptions and versions objects, should use `map()` to just include their names and uuids?  Also keep in mind that the uuids will be generated in previous code and will likely not be _in/on_ the versions or subscriptions that were passed into the function